.github/workflows/build.yml

```
@@ -0,0 +1,27 @@
+name: Build
+
+on:
+  pull_request:
+    types: [ labeled ]
+
+jobs:
+  build:
+    # only run a manual build when the PR comes from gatsby
```

Did you mean to write dependabot?
```suggestion
    # only run a manual build when the PR comes from dependabot
```

---

Yep, thanks.